plan/preprocess.go

```diff
@@ -218,11 +218,6 @@ func (p *preprocessor) checkDropDatabaseGrammar(stmt *ast.DropDatabaseStmt) {
 }
 
 func (p *preprocessor) checkCreateTableGrammar(stmt *ast.CreateTableStmt) {
-	if stmt.Table == nil {
```

Comment:
Any test cover this change?
Or are your sure `go test -cover` never run to this line?

Reply:
@mccxj Please address this comment.

Reply:
@mccxj Why remove this check ?

Reply:
parser.y shows ast.CreateTableStmt's Table will not be null.